spec/mailers/app_mailer_spec.rb

```diff
@@ -0,0 +1,8 @@
+require "rails_helper"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.